### ISSUE TITLE
[image_picker] 'Always' kill the app & debug session ...

### ISSUE THREAD

**Issue by imeDevelopers** [flutter/flutter#71686]
Everything was great with this version: 0.6.6+5

but now it kill the complete app every time i use the get Image by camera ..

current version is: 0.6.7+14

you should find solution for this or give us the ability to include the camera inside our app as widget and then we capture as we want ..

mobile: realme 5
Android: 10
win 10
flutter: 1.22.4
vs code

**Comment by TahaTesser** [comment]
Hi @imeDevelopers 
Just tried to reproduce with a similar device Realme 6 running Android 10 on the latest `stable` using the official [example](https://pub.dev/packages/image_picker/example).
Tried getting an image by the camera, bug session isn't killed. 

<details> 
<summary>logs</summary> 

```bash
[ +218 ms] Starting: Intent { act=android.intent.action.RUN flg=0x20000000 cmp=com.nevercode.triage/.MainActivity (has
extras) }
[   +1 ms] Waiting for observatory port to be available...
[+3821 ms] Observatory URL on device: http://127.0.0.1:35471/j_0zv8E4cMY=/
[   +1 ms] executing: /Users/tahatesser/Code/sdk/platform-tools/adb -s EUYTFEUSQSRGDA6D forward tcp:0 tcp:35471
[  +11 ms] 60309
[   +1 ms] Forwarded host port 60309 to device port 35471 for Observatory
[   +5 ms] Caching compiled dill
[ +105 ms] Connecting to service protocol: http://127.0.0.1:60309/j_0zv8E4cMY=/
[   +3 ms] Launching a Dart Developer Service (DDS) instance at http://127.0.0.1:0, connecting to VM service at
http://127.0.0.1:60309/j_0zv8E4cMY=/.
[ +698 ms] DDS is listening at http://127.0.0.1:60311/yC9szjtEPhg=/.
[  +86 ms] Successfully connected to service protocol: http://127.0.0.1:60309/j_0zv8E4cMY=/
[        ] Waiting for RMX2001 to report its views...
[  +16 ms] Waiting for RMX2001 to report its views... (completed in 16ms)
[  +21 ms] DevFS: Creating new filesystem on the device (null)
[ +107 ms] DevFS: Created new filesystem on the device
(file:///data/user/0/com.nevercode.triage/code_cache/image_picker_exampleDBYWMB/image_picker_example/)
[   +3 ms] Updating assets
[ +179 ms] Syncing files to device RMX2001...
[   +1 ms] <- reset
[        ] Compiling dart to kernel with 0 updated files
[   +2 ms] <- recompile package:triage/main.dart 237729eb-14bc-4b8f-8140-3a99ae35544f
[        ] <- 237729eb-14bc-4b8f-8140-3a99ae35544f
[ +148 ms] Updating files.
[        ] DevFS: Sync finished
[        ] Syncing files to device RMX2001... (completed in 155ms)
[   +1 ms] Synced 0.0MB.
[   +1 ms] <- accept
[  +17 ms] Connected to _flutterView/0x7095499020.
[   +4 ms] Flutter run key commands.
[   +1 ms] r Hot reload. 🔥🔥🔥
[   +1 ms] R Hot restart.
[        ] h Repeat this help message.
[        ] d Detach (terminate "flutter run" but leave application running).
[        ] c Clear the screen
[        ] q Quit (terminate the application on the device).
[        ] An Observatory debugger and profiler on RMX2001 is available at: http://127.0.0.1:60311/yC9szjtEPhg=/
[ +779 ms] I/evercode.triag(28421): ProcessProfilingInfo new_methods=523 is saved saved_to_disk=1
resolve_classes_delay=5000
[+158540 ms] V/PhoneWindow(28421): DecorView setVisiblity: visibility = 0, Parent = android.view.ViewRootImpl@2bca3ab,
this = DecorView@c57781c[MainActivity]
[  +48 ms] D/Surface (28421): Surface::connect(this=0x709546c000,api=1)
[   +3 ms] D/mali_winsys(28421): EGLint new_window_surface(egl_winsys_display *, void *, EGLSurface, EGLConfig,
egl_winsys_surface **, EGLBoolean) returns 0x3000
[   +1 ms] D/Surface (28421): Surface::setBufferCount(this=0x709546c000,bufferCount=4)
[        ] I/SurfaceView(28421): 176423456 visibleChanged:true -> SurfaceHolder.Callback.surfaceCreated
[        ] D/Surface (28421): Surface::connect(this=0x7003cc0000,api=1)
[   +3 ms] D/mali_winsys(28421): EGLint new_window_surface(egl_winsys_display *, void *, EGLSurface, EGLConfig,
egl_winsys_surface **, EGLBoolean) returns 0x3000
[   +5 ms] D/ViewRootImpl(28421): setSurfaceViewCreated, created:true
[        ] I/SurfaceView(28421): 176423456 surfaceChanged -- format=4 w=1080 h=2400 visibleChanged:true ->
SurfaceHolder.Callback.surfaceChanged
[        ] D/SurfaceView(28421): updateScreenMode w:1080 h:2400 ViewRoot w:1080 h:2400
[        ] D/SurfaceView(28421): try to resquest 60Hz
[        ] D/ViewRootImpl(28421): set screen refresh mode 1 due to SurfaceView-176423456
[   +2 ms] D/SurfaceView(28421): updateScreenMode w:1080 h:2400 ViewRoot w:1080 h:2400
[        ] D/SurfaceView(28421): try to resquest 60Hz
[  +11 ms] W/Gralloc3(28421): mapper 3.x is not supported
[   +5 ms] E/gralloc (28421): Arm Module v1.0
[        ] E/ion     (28421): ioctl c0044901 failed with code -1: Invalid argument
[        ] W/gralloc (28421): WARNING: internal format modifier bits not mutually exclusive. AFBC basic bit is always
set, so extended AFBC support bits must always be checked.
[ +238 ms] I/Choreographer(28421): Skipped 2 frames!  The application may be doing too much work on its main thread.
[+8452 ms] D/ColorViewRootUtil(28421): nav gesture mode swipeFromBottom ignore false downY 1875 mScreenHeight 2400
mScreenWidth 1080 mStatusBarHeight 54 globalScale 1.125 nav mode 3 event MotionEvent { action=ACTION_DOWN,
actionButton=0, id[0]=0, x[0]=898.0, y[0]=1875.0, toolType[0]=TOOL_TYPE_FINGER, buttonState=0, classification=NONE,
metaState=0, flags=0x0, edgeFlags=0x0, pointerCount=1, historySize=0, eventTime=345718979, downTime=345718979,
deviceId=3, source=0x1002, displayId=0 } rotation 0
[+2048 ms] D/ColorViewRootUtil(28421): nav gesture mode swipeFromBottom ignore false downY 2220 mScreenHeight 2400
mScreenWidth 1080 mStatusBarHeight 54 globalScale 1.125 nav mode 3 event MotionEvent { action=ACTION_DOWN,
actionButton=0, id[0]=0, x[0]=859.0, y[0]=2220.0, toolType[0]=TOOL_TYPE_FINGER, buttonState=0, classification=NONE,
metaState=0, flags=0x0, edgeFlags=0x0, pointerCount=1, historySize=0, eventTime=345721055, downTime=345721055,
deviceId=3, source=0x1002, displayId=0 } rotation 0
[ +610 ms] I/SurfaceView(28421): 176423456 visibleChanged:true -> SurfaceHolder.Callback.surfaceDestroyed
[   +9 ms] D/Surface (28421): Surface::disconnect(this=0x7003cc0000,api=1)
[        ] D/ViewRootImpl(28421): setSurfaceViewCreated, created:false
[        ] D/Surface (28421): Surface::disconnect(this=0x7003cc0000,api=-1)
[  +11 ms] D/SurfaceView(28421): updateScreenMode w:0 h:0 ViewRoot w:1080 h:2400
[        ] D/SurfaceView(28421): try to resquest 90Hz
[        ] D/ViewRootImpl(28421): set screen refresh mode 0 due to SurfaceView-176423456
[  +24 ms] D/Surface (28421): Surface::disconnect(this=0x709546c000,api=1)
[ +432 ms] V/PhoneWindow(28421): DecorView setVisiblity: visibility = 4, Parent = android.view.ViewRootImpl@2bca3ab,
this = DecorView@c57781c[MainActivity]
[  +19 ms] I/GED     (28421): ged_boost_gpu_freq, level 100, eOrigin 2, final_idx 27, oppidx_max 27, oppidx_min 0
[+6209 ms] D/ColorExSystemServiceHelper(28421): checkColorExSystemService className =
com.android.providers.media.MediaScannerService
[   +4 ms] V/PhoneWindow(28421): DecorView setVisiblity: visibility = 0, Parent = android.view.ViewRootImpl@2bca3ab,
this = DecorView@c57781c[MainActivity]
[  +29 ms] D/Surface (28421): Surface::connect(this=0x709546c000,api=1)
[   +3 ms] D/mali_winsys(28421): EGLint new_window_surface(egl_winsys_display *, void *, EGLSurface, EGLConfig,
egl_winsys_surface **, EGLBoolean) returns 0x3000
[        ] D/Surface (28421): Surface::setBufferCount(this=0x709546c000,bufferCount=4)
[   +4 ms] I/SurfaceView(28421): 176423456 visibleChanged:true -> SurfaceHolder.Callback.surfaceCreated
[        ] D/Surface (28421): Surface::connect(this=0x7003e76000,api=1)
[        ] D/mali_winsys(28421): EGLint new_window_surface(egl_winsys_display *, void *, EGLSurface, EGLConfig,
egl_winsys_surface **, EGLBoolean) returns 0x3000
[  +13 ms] D/ViewRootImpl(28421): setSurfaceViewCreated, created:true
[        ] I/SurfaceView(28421): 176423456 surfaceChanged -- format=4 w=1080 h=2400 visibleChanged:true ->
SurfaceHolder.Callback.surfaceChanged
[        ] D/SurfaceView(28421): updateScreenMode w:1080 h:2400 ViewRoot w:1080 h:2400
[        ] D/SurfaceView(28421): try to resquest 60Hz
[        ] D/ViewRootImpl(28421): set screen refresh mode 1 due to SurfaceView-176423456
[   +3 ms] D/SurfaceView(28421): updateScreenMode w:1080 h:2400 ViewRoot w:1080 h:2400
[        ] D/SurfaceView(28421): try to resquest 60Hz
[  +38 ms] I/Choreographer(28421): Skipped 3 frames!  The application may be doing too much work on its main thread.
[  +13 ms] D/SurfaceView(28421): updateScreenMode w:1080 h:2400 ViewRoot w:1080 h:2400
[        ] D/SurfaceView(28421): try to resquest 60Hz
[  +28 ms] D/IAtlas  (28421): IAtlas::init CallingPid 0
[        ] D/IAtlas  (28421): IAtlas::init this 0x70038222a0
[        ] D/skia    (28421): SkJpegCodec::onGetPixels +
[  +24 ms] D/skia    (28421): stream getLength() not supported, use temp buffer for loading stream, buffer addr
0x6f5a7046c0 length 3270185
[   +1 ms] W/Binder:28421_4(28421): type=1400 audit(0.0:150646): avc: denied { search } for name="ppm" dev="proc"
ino=4026531942 scontext=u:r:untrusted_app_27:s0:c230,c257,c512,c768 tcontext=u:object_r:proc_ppm:s0 tclass=dir
permissive=0
[   +2 ms] D/libjpeg-alpha(28421): Tile Decoder (#thread:4, tile size:1024x1152, alignment:256x16)
[        ] D/libjpeg-alpha(28421): Huffman Builder run in subthread
[  +82 ms] D/skia    (28421): SkJpegCodec::onGetPixels -
[+30557 ms] I/SurfaceView(28421): 176423456 visibleChanged:true -> SurfaceHolder.Callback.surfaceDestroyed
[   +3 ms] D/Surface (28421): Surface::disconnect(this=0x7003e76000,api=1)
[   +4 ms] D/ViewRootImpl(28421): setSurfaceViewCreated, created:false
[        ] D/Surface (28421): Surface::disconnect(this=0x7003e76000,api=-1)
[        ] D/SurfaceView(28421): updateScreenMode w:0 h:0 ViewRoot w:1080 h:2400
[        ] D/SurfaceView(28421): try to resquest 90Hz
[        ] D/ViewRootImpl(28421): set screen refresh mode 0 due to SurfaceView-176423456
[  +29 ms] I/Choreographer(28421): Skipped 1 frames!  The application may be doing too much work on its main thread.
[   +2 ms] I/GED     (28421): ged_boost_gpu_freq, level 100, eOrigin 2, final_idx 27, oppidx_max 27, oppidx_min 0
[+4542 ms] D/Surface (28421): Surface::disconnect(this=0x709546c000,api=1)
[  +20 ms] V/PhoneWindow(28421): DecorView setVisiblity: visibility = 4, Parent = android.view.ViewRootImpl@2bca3ab,
this = DecorView@c57781c[MainActivity]
[+43683 ms] V/PhoneWindow(28421): DecorView setVisiblity: visibility = 0, Parent = android.view.ViewRootImpl@2bca3ab,
this = DecorView@c57781c[MainActivity]
[  +25 ms] D/Surface (28421): Surface::connect(this=0x7003e75000,api=1)
[        ] D/mali_winsys(28421): EGLint new_window_surface(egl_winsys_display *, void *, EGLSurface, EGLConfig,
egl_winsys_surface **, EGLBoolean) returns 0x3000
[   +1 ms] D/Surface (28421): Surface::setBufferCount(this=0x7003e75000,bufferCount=4)
[   +3 ms] I/SurfaceView(28421): 176423456 visibleChanged:true -> SurfaceHolder.Callback.surfaceCreated
[   +1 ms] D/Surface (28421): Surface::connect(this=0x7003ce0000,api=1)
[   +4 ms] D/mali_winsys(28421): EGLint new_window_surface(egl_winsys_display *, void *, EGLSurface, EGLConfig,
egl_winsys_surface **, EGLBoolean) returns 0x3000
[   +1 ms] D/ViewRootImpl(28421): setSurfaceViewCreated, created:true
[        ] I/SurfaceView(28421): 176423456 surfaceChanged -- format=4 w=1080 h=2400 visibleChanged:true ->
SurfaceHolder.Callback.surfaceChanged
[        ] D/SurfaceView(28421): updateScreenMode w:1080 h:2400 ViewRoot w:1080 h:2400
[        ] D/SurfaceView(28421): try to resquest 60Hz
[        ] D/ViewRootImpl(28421): set screen refresh mode 1 due to SurfaceView-176423456
[        ] D/SurfaceView(28421): updateScreenMode w:1080 h:2400 ViewRoot w:1080 h:2400
[        ] D/SurfaceView(28421): try to resquest 60Hz
[  +17 ms] I/Choreographer(28421): Skipped 1 frames!  The application may be doing too much work on its main thread.
[+2909 ms] D/ColorViewRootUtil(28421): nav gesture mode swipeFromBottom ignore false downY 1834 mScreenHeight 2400
mScreenWidth 1080 mStatusBarHeight 54 globalScale 1.125 nav mode 3 event MotionEvent { action=ACTION_DOWN,
actionButton=0, id[0]=0, x[0]=971.0, y[0]=1834.0, toolType[0]=TOOL_TYPE_FINGER, buttonState=0, classification=NONE,
metaState=0, flags=0x0, edgeFlags=0x0, pointerCount=1, historySize=0, eventTime=345810442, downTime=345810442,
deviceId=3, source=0x1002, displayId=0 } rotation 0
[+1135 ms] D/ColorViewRootUtil(28421): nav gesture mode swipeFromBottom ignore false downY 2268 mScreenHeight 2400
mScreenWidth 1080 mStatusBarHeight 54 globalScale 1.125 nav mode 3 event MotionEvent { action=ACTION_DOWN,
actionButton=0, id[0]=0, x[0]=807.0, y[0]=2268.0, toolType[0]=TOOL_TYPE_FINGER, buttonState=0, classification=NONE,
metaState=0, flags=0x0, edgeFlags=0x0, pointerCount=1, historySize=0, eventTime=345811576, downTime=345811576,
deviceId=3, source=0x1002, displayId=0 } rotation 0
[ +699 ms] I/SurfaceView(28421): 176423456 visibleChanged:true -> SurfaceHolder.Callback.surfaceDestroyed
[  +10 ms] D/Surface (28421): Surface::disconnect(this=0x7003ce0000,api=1)
[   +4 ms] D/ViewRootImpl(28421): setSurfaceViewCreated, created:false
[        ] D/Surface (28421): Surface::disconnect(this=0x7003ce0000,api=-1)
[        ] D/SurfaceView(28421): updateScreenMode w:0 h:0 ViewRoot w:1080 h:2400
[        ] D/SurfaceView(28421): try to resquest 90Hz
[        ] D/ViewRootImpl(28421): set screen refresh mode 0 due to SurfaceView-176423456
[  +16 ms] D/Surface (28421): Surface::disconnect(this=0x7003e75000,api=1)
[  +21 ms] I/GED     (28421): ged_boost_gpu_freq, level 100, eOrigin 2, final_idx 27, oppidx_max 27, oppidx_min 0
[  +14 ms] V/PhoneWindow(28421): DecorView setVisiblity: visibility = 4, Parent = android.view.ViewRootImpl@2bca3ab,
this = DecorView@c57781c[MainActivity]
[  +25 ms] I/Choreographer(28421): Skipped 3 frames!  The application may be doing too much work on its main thread.
[+3492 ms] D/ColorExSystemServiceHelper(28421): checkColorExSystemService className =
com.android.providers.media.MediaScannerService
[   +1 ms] V/PhoneWindow(28421): DecorView setVisiblity: visibility = 0, Parent = android.view.ViewRootImpl@2bca3ab,
this = DecorView@c57781c[MainActivity]
[  +17 ms] D/Surface (28421): Surface::connect(this=0x7003cdf000,api=1)
[   +9 ms] D/mali_winsys(28421): EGLint new_window_surface(egl_winsys_display *, void *, EGLSurface, EGLConfig,
egl_winsys_surface **, EGLBoolean) returns 0x3000
[        ] D/Surface (28421): Surface::setBufferCount(this=0x7003cdf000,bufferCount=4)
[        ] I/SurfaceView(28421): 176423456 visibleChanged:true -> SurfaceHolder.Callback.surfaceCreated
[        ] D/Surface (28421): Surface::connect(this=0x709545b000,api=1)
[   +3 ms] D/mali_winsys(28421): EGLint new_window_surface(egl_winsys_display *, void *, EGLSurface, EGLConfig,
egl_winsys_surface **, EGLBoolean) returns 0x3000
[  +24 ms] D/ViewRootImpl(28421): setSurfaceViewCreated, created:true
[        ] I/SurfaceView(28421): 176423456 surfaceChanged -- format=4 w=1080 h=2400 visibleChanged:true ->
SurfaceHolder.Callback.surfaceChanged
[        ] D/SurfaceView(28421): updateScreenMode w:1080 h:2400 ViewRoot w:1080 h:2400
[        ] D/SurfaceView(28421): try to resquest 60Hz
[        ] D/ViewRootImpl(28421): set screen refresh mode 1 due to SurfaceView-176423456
[   +7 ms] D/SurfaceView(28421): updateScreenMode w:1080 h:2400 ViewRoot w:1080 h:2400
[        ] D/SurfaceView(28421): try to resquest 60Hz
[  +45 ms] I/Choreographer(28421): Skipped 3 frames!  The application may be doing too much work on its main thread.
[  +13 ms] D/SurfaceView(28421): updateScreenMode w:1080 h:2400 ViewRoot w:1080 h:2400
[        ] D/SurfaceView(28421): try to resquest 60Hz
[  +19 ms] D/IAtlas  (28421): IAtlas::init CallingPid 0
[   +2 ms] D/IAtlas  (28421): IAtlas::init this 0x6fff0f9680
[        ] D/skia    (28421): SkJpegCodec::onGetPixels +
[  +10 ms] D/skia    (28421): stream getLength() not supported, use temp buffer for loading stream, buffer addr
0x6f549c6e80 length 2720218
[        ] W/Binder:28421_3(28421): type=1400 audit(0.0:150680): avc: denied { search } for name="ppm" dev="proc"
ino=4026531942 scontext=u:r:untrusted_app_27:s0:c230,c257,c512,c768 tcontext=u:object_r:proc_ppm:s0 tclass=dir
permissive=0
[        ] D/libjpeg-alpha(28421): Tile Decoder (#thread:4, tile size:1024x1152, alignment:256x16)
[        ] D/libjpeg-alpha(28421): Huffman Builder run in subthread
[        ] W/Binder:28421_3(28421): type=1400 audit(0.0:150681): avc: denied { search } for name="ppm" dev="proc"
ino=4026531942 scontext=u:r:untrusted_app_27:s0:c230,c257,c512,c768 tcontext=u:object_r:proc_ppm:s0 tclass=dir
permissive=0
[  +70 ms] D/skia    (28421): SkJpegCodec::onGetPixels -
[+1438 ms] D/ColorViewRootUtil(28421): nav gesture mode swipeFromBottom ignore false downY 1844 mScreenHeight 2400
mScreenWidth 1080 mStatusBarHeight 54 globalScale 1.125 nav mode 3 event MotionEvent { action=ACTION_DOWN,
actionButton=0, id[0]=0, x[0]=959.0, y[0]=1844.0, toolType[0]=TOOL_TYPE_FINGER, buttonState=0, classification=NONE,
metaState=0, flags=0x0, edgeFlags=0x0, pointerCount=1, historySize=0, eventTime=345817529, downTime=345817529,
deviceId=3, source=0x1002, displayId=0 } rotation 0
[+1323 ms] D/ColorViewRootUtil(28421): nav gesture mode swipeFromBottom ignore false downY 2230 mScreenHeight 2400
mScreenWidth 1080 mStatusBarHeight 54 globalScale 1.125 nav mode 3 event MotionEvent { action=ACTION_DOWN,
actionButton=0, id[0]=0, x[0]=810.0, y[0]=2230.0, toolType[0]=TOOL_TYPE_FINGER, buttonState=0, classification=NONE,
metaState=0, flags=0x0, edgeFlags=0x0, pointerCount=1, historySize=0, eventTime=345818851, downTime=345818851,
deviceId=3, source=0x1002, displayId=0 } rotation 0
[ +703 ms] I/SurfaceView(28421): 176423456 visibleChanged:true -> SurfaceHolder.Callback.surfaceDestroyed
[  +13 ms] D/Surface (28421): Surface::disconnect(this=0x709545b000,api=1)
[        ] D/ViewRootImpl(28421): setSurfaceViewCreated, created:false
[        ] D/Surface (28421): Surface::disconnect(this=0x709545b000,api=-1)
[        ] D/SurfaceView(28421): updateScreenMode w:0 h:0 ViewRoot w:1080 h:2400
[        ] D/SurfaceView(28421): try to resquest 90Hz
[        ] D/ViewRootImpl(28421): set screen refresh mode 0 due to SurfaceView-176423456
[   +4 ms] D/Surface (28421): Surface::disconnect(this=0x7003cdf000,api=1)
[  +18 ms] I/GED     (28421): ged_boost_gpu_freq, level 100, eOrigin 2, final_idx 27, oppidx_max 27, oppidx_min 0
[   +5 ms] V/PhoneWindow(28421): DecorView setVisiblity: visibility = 4, Parent = android.view.ViewRootImpl@2bca3ab,
this = DecorView@c57781c[MainActivity]
[  +15 ms] I/Choreographer(28421): Skipped 1 frames!  The application may be doing too much work on its main thread.
[+2673 ms] D/ColorExSystemServiceHelper(28421): checkColorExSystemService className =
com.android.providers.media.MediaScannerService
[        ] V/PhoneWindow(28421): DecorView setVisiblity: visibility = 0, Parent = android.view.ViewRootImpl@2bca3ab,
this = DecorView@c57781c[MainActivity]
[  +27 ms] D/Surface (28421): Surface::connect(this=0x7003e75000,api=1)
[   +3 ms] D/mali_winsys(28421): EGLint new_window_surface(egl_winsys_display *, void *, EGLSurface, EGLConfig,
egl_winsys_surface **, EGLBoolean) returns 0x3000
[   +1 ms] D/Surface (28421): Surface::setBufferCount(this=0x7003e75000,bufferCount=4)
[        ] I/SurfaceView(28421): 176423456 visibleChanged:true -> SurfaceHolder.Callback.surfaceCreated
[        ] D/Surface (28421): Surface::connect(this=0x7003e8c000,api=1)
[        ] D/mali_winsys(28421): EGLint new_window_surface(egl_winsys_display *, void *, EGLSurface, EGLConfig,
egl_winsys_surface **, EGLBoolean) returns 0x3000
[        ] D/ViewRootImpl(28421): setSurfaceViewCreated, created:true
[        ] I/SurfaceView(28421): 176423456 surfaceChanged -- format=4 w=1080 h=2400 visibleChanged:true ->
SurfaceHolder.Callback.surfaceChanged
[        ] D/SurfaceView(28421): updateScreenMode w:1080 h:2400 ViewRoot w:1080 h:2400
[        ] D/SurfaceView(28421): try to resquest 60Hz
[   +9 ms] D/ViewRootImpl(28421): set screen refresh mode 1 due to SurfaceView-176423456
[   +2 ms] D/IAtlas  (28421): IAtlas::init CallingPid 0
[        ] D/SurfaceView(28421): updateScreenMode w:1080 h:2400 ViewRoot w:1080 h:2400
[        ] D/SurfaceView(28421): try to resquest 60Hz
[        ] D/IAtlas  (28421): IAtlas::init this 0x70038222a0
[  +28 ms] D/skia    (28421): SkJpegCodec::onGetPixels +
[  +18 ms] D/skia    (28421): stream getLength() not supported, use temp buffer for loading stream, buffer addr
0x6f59e01c00 length 3596786
[   +2 ms] W/Binder:28421_4(28421): type=1400 audit(0.0:150703): avc: denied { search } for name="ppm" dev="proc"
ino=4026531942 scontext=u:r:untrusted_app_27:s0:c230,c257,c512,c768 tcontext=u:object_r:proc_ppm:s0 tclass=dir
permissive=0
[        ] W/Binder:28421_4(28421): type=1400 audit(0.0:150704): avc: denied { search } for name="ppm" dev="proc"
ino=4026531942 scontext=u:r:untrusted_app_27:s0:c230,c257,c512,c768 tcontext=u:object_r:proc_ppm:s0 tclass=dir
permissive=0
[        ] D/libjpeg-alpha(28421): Tile Decoder (#thread:4, tile size:1024x1152, alignment:256x16)
[        ] D/libjpeg-alpha(28421): Huffman Builder run in subthread
[  +22 ms] I/Choreographer(28421): Skipped 6 frames!  The application may be doing too much work on its main thread.
[  +13 ms] D/SurfaceView(28421): updateScreenMode w:1080 h:2400 ViewRoot w:1080 h:2400
[        ] D/SurfaceView(28421): try to resquest 60Hz
[  +79 ms] D/skia    (28421): SkJpegCodec::onGetPixels -
[+28110 ms] D/ColorViewRootUtil(28421): nav gesture mode swipeFromBottom ignore true downY 2319 mScreenHeight 2400
mScreenWidth 1080 mStatusBarHeight 54 globalScale 1.125 nav mode 3 event MotionEvent { action=ACTION_DOWN,
actionButton=0, id[0]=0, x[0]=1009.0, y[0]=2319.0, toolType[0]=TOOL_TYPE_FINGER, buttonState=0, classification=NONE,
metaState=0, flags=0x0, edgeFlags=0x0, pointerCount=1, historySize=0, eventTime=345850618, downTime=345850618,
deviceId=3, source=0x1002, displayId=0 } rotation 0
[  +53 ms] D/ColorViewRootUtil(28421): do not ignore inject event MotionEvent:MotionEvent { action=ACTION_DOWN,
actionButton=0, id[0]=0, x[0]=1009.0, y[0]=2319.0, toolType[0]=TOOL_TYPE_FINGER, buttonState=0, classification=NONE,
metaState=0, flags=0x0, edgeFlags=0x0, pointerCount=1, historySize=0, eventTime=345850618, downTime=345850617,
deviceId=3, source=0x1002, displayId=0 }
[+1221 ms] D/ColorViewRootUtil(28421): nav gesture mode swipeFromBottom ignore true downY 2375 mScreenHeight 2400
mScreenWidth 1080 mStatusBarHeight 54 globalScale 1.125 nav mode 3 event MotionEvent { action=ACTION_DOWN,
actionButton=0, id[0]=0, x[0]=619.0, y[0]=2375.0, toolType[0]=TOOL_TYPE_FINGER, buttonState=0, classification=NONE,
metaState=0, flags=0x0, edgeFlags=0x0, pointerCount=1, historySize=0, eventTime=345851894, downTime=345851894,
deviceId=3, source=0x1002, displayId=0 } rotation 0
[ +522 ms] I/SurfaceView(28421): 176423456 visibleChanged:true -> SurfaceHolder.Callback.surfaceDestroyed
[   +8 ms] D/Surface (28421): Surface::disconnect(this=0x7003e8c000,api=1)
[   +4 ms] D/ViewRootImpl(28421): setSurfaceViewCreated, created:false
[        ] D/Surface (28421): Surface::disconnect(this=0x7003e8c000,api=-1)
[        ] D/SurfaceView(28421): updateScreenMode w:0 h:0 ViewRoot w:1080 h:2400
[        ] D/SurfaceView(28421): try to resquest 90Hz
[        ] D/ViewRootImpl(28421): set screen refresh mode 0 due to SurfaceView-176423456
[        ] D/Surface (28421): Surface::disconnect(this=0x7003e75000,api=1)
[  +12 ms] V/PhoneWindow(28421): DecorView setVisiblity: visibility = 4, Parent = android.view.ViewRootImpl@2bca3ab,
this = DecorView@c57781c[MainActivity]
[   +8 ms] I/GED     (28421): ged_boost_gpu_freq, level 100, eOrigin 2, final_idx 27, oppidx_max 27, oppidx_min 0

```

</details>

![Screenshot_2020-12-04-13-47-56-64](https://user-images.githubusercontent.com/48603081/101139500-a4bec900-3637-11eb-8768-8ee4d6747e51.jpg)


<details> 
<summary>flutter doctor -v</summary> 

```bash
[✓] Flutter (Channel stable, 1.22.4, on macOS 11.0.1 20B29 darwin-x64, locale
    en-GB)
    • Flutter version 1.22.4 at /Users/tahatesser/Code/flutter_stable
    • Framework revision 1aafb3a8b9 (3 weeks ago), 2020-11-13 09:59:28 -0800
    • Engine revision 2c956a31c0
    • Dart version 2.10.4

 
[✓] Android toolchain - develop for Android devices (Android SDK version 30.0.2)
    • Android SDK at /Users/tahatesser/Code/sdk
    • Platform android-30, build-tools 30.0.2
    • ANDROID_HOME = /Users/tahatesser/Code/sdk
    • Java binary at: /Applications/Android
      Studio.app/Contents/jre/jdk/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build
      1.8.0_242-release-1644-b3-6915495)
    • All Android licenses accepted.

[✓] Xcode - develop for iOS and macOS (Xcode 12.2)
    • Xcode at /Volumes/Extreme/Xcode.app/Contents/Developer
    • Xcode 12.2, Build version 12B45b
    • CocoaPods version 1.10.0

[!] Android Studio (version 4.1)
    • Android Studio at /Applications/Android Studio.app/Contents
    ✗ Flutter plugin not installed; this adds Flutter specific functionality.
    ✗ Dart plugin not installed; this adds Dart specific functionality.
    • Java version OpenJDK Runtime Environment (build
      1.8.0_242-release-1644-b3-6915495)

[✓] VS Code (version 1.51.1)
    • VS Code at /Applications/Visual Studio Code.app/Contents
    • Flutter extension version 3.16.0

[✓] Connected device (2 available)
    • RMX2001 (mobile)     • EUYTFEUSQSRGDA6D          • android-arm64 • Android
      10 (API 29)
    • Taha’s iPad (mobile) • 00008020-000255113EE8402E • ios           • iOS
      14.2

! Doctor found issues in 1 category.
```

</details>


Can you please provide your `flutter doctor -v` your `flutter run --verbose` while getting an image from camera and a complete reproducible minimal code sample
Thank you

**Comment by pedromassangocode** [comment]
Without additional information, we are unfortunately not sure how to resolve this issue. 
We are therefore reluctantly going to close this bug for now. 
Please don't hesitate to comment on the bug if you have any more information for us; we will reopen it right away!  
Thanks for your contribution.

Could everyone who still has this problem please file a new issue with the exact description of what happens, logs, and the output of `flutter doctor -v`.
All system setups can be slightly different, so it's always better to open new issues and reference related issues.
